ssas/systems.go

```diff
@@ -698,21 +663,21 @@ func (system *System) GetIps(ctx context.Context, trackingID string) ([]IP, erro
 }
 
 // DeleteIP soft-deletes an IP associated with a specific system
-func (system *System) DeleteIP(ctx context.Context, ipID string, trackingID string) error {
+func (system *System) DeleteIP(ctx context.Context, ipID string) error {
 	var (
 		ip  IP
 		err error
 	)
 
-	regEvent := Event{Op: "DeleteIP", TrackingID: trackingID, Help: "calling from ssas.DeleteIP()"}
+	regEvent := Event{Op: "DeleteIP", Help: "calling from ssas.DeleteIP()"}
 	OperationStarted(regEvent)
 
 	// Find IP to delete
 	err = Connection.WithContext(ctx).First(&ip, "system_id = ? AND id = ?", system.ID, ipID).Error
 	if err != nil {
 		regEvent.Help = fmt.Sprintf("Unable to find IP address with ID %s: %s", ipID, err)
 		OperationFailed(regEvent)
```

Comment:
Is there a different ticket that tracks the rest of the OperationFailed, OperationStarted, etc?